src/sites/upload.rs

```diff
@@ -1,82 +0,0 @@
-use indicatif::{ProgressBar, ProgressStyle};
```

Comment:
moved this logic to `bulk::batch_keys_values` since it was no longer specific to sites anyway